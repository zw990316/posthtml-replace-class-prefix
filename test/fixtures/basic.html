<html> 
    <div class="el-table">
        <a class="el-tree">
            <i >
                <span class="el-iicon"></span>
            </i>
        </a>
    </div>
    <div class="el-table">
        <a class="el-tree">
            <i >
                <span class="el-iicon"></span>
            </i>
        </a>
    </div>
    <div class="el-table">
        <a class="el-tree">
            <i >
                <span class="el-iicon"></span>
            </i>
        </a>
    </div>
</html>
