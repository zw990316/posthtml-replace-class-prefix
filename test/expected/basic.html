<html>
    <div class="gx-table">
        <a class="gx-tree">
            <i>
                <span class="gx-iicon"></span>
            </i>
        </a>
    </div>
    <div class="gx-table">
        <a class="gx-tree">
            <i>
                <span class="gx-iicon"></span>
            </i>
        </a>
    </div>
    <div class="gx-table">
        <a class="gx-tree">
            <i>
                <span class="gx-iicon"></span>
            </i>
        </a>
    </div>
</html>
